src/sentry/features/temporary.py

```diff
@@ -186,6 +186,7 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:profiling-global-suspect-functions", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
     manager.add("organizations:profiling-summary-redesign", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
     manager.add("organizations:profiling-using-transactions", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
+    manager.add("organizations:continuous-profiling", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
```

Comment:
you need to set a default for this feature flag in https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L1452